packages/server/lib/reporter.coffee

```diff
@@ -40,8 +40,8 @@ getParentTitle = (runnable, titles) ->
 createSuite = (obj, parent) ->
   suite = new Mocha.Suite(obj.title, {})
   suite.parent = parent if parent
-  if obj.file
-    console.log('has file:', obj.file)
+  # if obj.file
+    # console.log('has file:', obj.file)
```

Comment:
Any reason this is left commented out? We do not advise leaving commented out code that is no longer in use or necessary. Why was this removed?

Reply:
I can certainly remove them, I'm not sure why they were there to start. They added the logs to the circleci test you observed (see earlier comments), because once file was being passed in this logic was involved. Since I was unsure of the purpose of these lines of code, I commented them out, to gain an understanding of they were meant to be in there for some reason... If it's cool with you, I'll just remove the lines of code.